src/template-tags/tickets.php

```diff
@@ -1600,61 +1600,6 @@ function tribe_tickets_rsvp_new_views_is_enabled() {
 	return apply_filters( 'tribe_tickets_rsvp_new_views_is_enabled', $enabled );
 }
 
-if ( ! function_exists( 'tribe_tickets_ar_field_is_required' ) ) {
-	/**
-	 * Check if the AR field is required.
-	 *
-	 * @since 4.12.3
-	 *
-	 * @param object $field The field object.
-	 *
-	 * @return bool True if is required
-	 */
-	function tribe_tickets_ar_field_is_required( $field ) {
-		return isset( $field->required ) && 'on' === $field->required;
-	}
-}
-
-if ( ! function_exists( 'tribe_tickets_ar_field_name' ) ) {
-	/**
-	 * Build the AR field name.
-	 *
-	 * @since 4.12.3
-	 *
-	 * @param int    $ticket_id  The ticket ID.
-	 * @param string $field_slug The field slug.
-	 *
-	 * @return string The AR field name.
-	 */
-	function tribe_tickets_ar_field_name( $ticket_id, $field_slug ) {
```

Comment:
Instead of removing, should we deprecate them and call the new functions in ET+?

Reply:
I introduced them when I started working on templates for RSVP, so they were never used outside of the templates I just moved to ET+. It made sense to remove them completely as we never used them/introduced them into code that was actually "production".